articles/libraries/auth0js/v9/index.md

```diff
@@ -424,6 +424,8 @@ Note that `checkSession()` triggers any [rules](/rules) you may have set up, so
 
 The actual redirect to `/authorize` happens inside an iframe, so it will not reload your application or redirect away from it.
 
+However, the browser **must** have third-party cookies enabled (or there cannot be something that is [functionality equivalent to disabling third-party cookies, such as Safari's ITP feature](/api-auth/token-renewal-in-safari)). Otherwise, **checkSession()** is unable to access the current user's session (making it impossible to obtain a new token without displaying anything to the user).
+
```

Comment:
What about

However, the browser **must** have third-party cookies enabled. Otherwise, **checkSession()** is unable to access the current user's session (making it impossible to obtain a new token without displaying anything to the user). The same will happen if users have [Safari's ITP enabled](/api-auth/token-renewal-in-safari).